### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by rivasbolinga**
Firs pull request. Creating branches. .ignore file and descriptive commits

### CODE REVIEW

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @rivasbolinga 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Pull request created ✔️

## Required Changes ♻️
- [ ] Kindly add the content of your linter.yml file to make your linters work and become green
- [ ] Consider adding `node_modules` and `test.md` in `.gitignore` file.
- [ ] Also consider adding the content of the HTML file and CSS file that includes one class with styles for your header (make it in your favorite color).
- [ ] Also consider using the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make a descriptive readme for your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rivasbolinga** [comment]
Hi @omarsalem7 Omar, thank you so much for your review,
 I came with an  issue:
I have already done all that you said, and I can see it in my browser, in my VS code and in my local folder but I don't know why I don't see it in Github even if I did commit it, push and even pull it. What can be the problem?
Regards

**Comment by rivasbolinga** [comment]
@omarsalem7 Hi I think i fixed the problem, What 
should I do  next?

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @rivasbolinga, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are passing 🟢 👍

 ## Required Changes ♻️

Good job! but it's recommended to add all of the linter configuration files, your repository is missing [three of them](https://github.com/microverseinc/linters-config/tree/master/html-css-js), please make sure to add them as well.

image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rivasbolinga** [comment]
Hi @aimalamiri , 
First, I would like to thank you for your review and your time. I have read your review and I have some doubts:

-Since I am just using HTML and CSS in this project, and also the Microverse requierements are "Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.", should I still add the rest of the linters?
-What do you want to mean when you say "Check the comments under the review."? 
I look forward to hearing from you
Cheers

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+- GitHub: [@rivasbolinga](https://github.com/rivasbolinga)
+- Twitter: [@AndreaRivasPal](https://twitter.com/AndreaRivasPal)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (OyePriscilla) [inline README.md:50]: - [ ] [optional] You can as well add the link to your issues here. 👍

**Review by OyePriscilla — APPROVED** [review]
## Hi @rivasbolinga 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

Highlights
Good job for using the right GitHub Flow 💯
Awesome file organization 🗃️
Commits are descriptive and meaningful 👍
Linters are passing 🟢 👍

## Optional suggestions
Kindly take note of the optional suggestions from the inline comments below, I'm sorry that the previous reviewers did not raise it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**